source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -383,6 +389,60 @@ namespace
         printAssistedCapture3DResults(allDurations);
     }
 
+    
```

Comment:
You should implement a for loop numFrames times for benchmarking copyData, see how we do it here:

https://github.com/zivid/zivid-cpp-samples/blob/223e70cc7a2605cfd38035422e1f5f5fd647917d/source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp#L448-L459

Reply:
After I check SE, General post, I tried your comment([link](https://teams.microsoft.com/l/message/19:e665fe301ba546c89741ed5b3ec707f4@thread.skype/1637925231993?tenantId=c3314a3e-055f-475e-b454-0d0876554b46&groupId=88796bff-4534-4964-8794-03a696974ebf&parentMessageId=1637910687845&teamName=Zivid&channelName=SE%2C%20General&createdTime=1637925231993)).
It was also not like my imagine.
Loop 1time (median, mean)
Suggest settings time:        56.331 ms    56.331 ms
2times
Suggest settings time:        43.478 ms    43.478 ms
3times
Suggest settings time:        21.558 ms    30.126 ms
5times
Suggest settings time:        6.205 ms     21.331 ms
10times
Suggest settings time:        6.177 ms     13.514 ms